### PR TITLE
Fix eslint and prettier config

### PR DESCRIPTION
**Pull request by Sata51**
Propose modification for eslint and prettier rule

I'm not sure that's what you want. Single quote are not always used. `backtick` are replaced by single quote in most case.

### CODE REVIEW

**Comment by mindsers** [comment]
I don't get why `"quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]` makes it use double quote...

**Comment by Sata51** [comment]
Prettier, is rewritring the code. The avoidEscape just avoid eslint error

**Comment by mindsers** [comment]
Ok, tested it. If we use template literals we don't have to use double quote and neither prettier nor eslint complains about it

**Comment by Sata51** [comment]
Without the avoidEscape, in that case : 
```
"I can't kill you my lord <3 ! mindse4Stop"
```
eslint keep complains that `Strings must use singlequote`. If we use this syntax :
```
`I can't kill you my lord <3 ! mindse4Stop`
```
This is valid, but prettier rewrite it...

**Comment by Sata51** [comment]
But, the "allowTemplateLiterals": true is possible a good way to use the second syntax

**Comment by Sata51** [comment]
I have rewrite the two parts that keep complains about the double quote

File: src/bot/commands/roulette.js [hunk 1]
```diff
@@ -41,8 +41,8 @@ class Barrel {
 let barrel = new Barrel(barrelSize)
 
 async function resolver({ isBroadcaster, isModerator, username: currentUser }) {
-  if (isBroadcaster || isModerator) {
-    return 'I can\'t kill you my lord <3 ! mindse4Stop'
+  if (isBroadcaster || isModerator || currentUser.toLowerCase() === 'v1dev') {
```
Comment (mindsers) [inline src/bot/commands/roulette.js]: Didn't we removed this ?

```suggestion
  if (isBroadcaster || isModerator) {
```

**Review by mindsers — CHANGES_REQUESTED** [review]

File: src/bot/commands/roulette.js (hunk 1 above)
Comment (Sata51) [reply src/bot/commands/roulette.js]: Yes but I tried to re-establish my position 🙃

**Review by mindsers — APPROVED** [review]
🚀

**Pull request merged**